src/styles/card.scss

```diff
@@ -0,0 +1,130 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: 16px;
+
+  padding: 32px 16px 16px;
+  width: 200px;
+  box-sizing: border-box;
+
+  border-radius: 5px;
+  border: 1px solid $accent-border-color;
+  background: $main-white;
+
+
+
+  &__info-content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 40px;
+  }
+
+  &__product-image {
+    display: block;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__text-info {
+    height: auto;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__product-title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-accent-color;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 18px;
+    color: $secondary-color;
+  }
+
+  &__rate {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__revievs-count {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $main-accent-color;
+  }
+
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-top: 8px;
+  }
+
+  &__price-title {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: $secondary-color;
+  }
+
+  &__price-count {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $main-accent-color;
+  }
+
+  &__buy-link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-decoration: none;
+    font-size: 14px;
+    font-weight: 700;
+    text-transform: uppercase;
+    background-color: $accent-color;
+    line-height: 16px;
+    height: 40px;
+    text-align: center;
+    color: $main-white;
+    border-radius: 5px;
+    box-sizing: border-box;
+
+    &:hover {
+      background-color: $main-white;
+      color: $accent-color;
+      border: 1px solid $accent-color;
+    }
+  }
+}
+
+
+
```

Comment:
too many lines...

good practice is to leave 1 line at the end of the file (please do this fix in each your file)